lib/asciidoctor/opal_ext/kernel.rb

```diff
@@ -1,6 +1,8 @@
 module Kernel
+  # Redefine Kernel#freeze to suppress "not supported" warning message from Opal.
+  # TODO Remove once https://github.com/opal/opal/issues/1253 is resolved.
```

Comment:
I think we can now remove this code because https://github.com/opal/opal/issues/1253 is "resolved" (Elia made some changes, described here: https://github.com/opal/opal/pull/1345#issuecomment-200085541).
What we need to do is to set `Opal.config.unsupported_features_severity = 'ignore'` to suppress the "not supported" warning message.